pbs_light.go

```diff
@@ -142,6 +143,64 @@ func getAccountMetrics(id string) *AccountMetrics {
 	return am
 }
 
+type cookieSyncRequest struct {
+	UUID    string   `json:"uuid"`
+	Bidders []string `json"bidders"`
+}
+
+type cookieSyncResponse struct {
+	UUID         string           `json:"uuid"`
+	Status       string           `json:"status"`
+	BidderStatus []*pbs.PBSBidder `json:"bidder_status"`
+}
+
+func cookieSync(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
```

Comment:
Does it make sense to start putting each request handler in it's own file? `pbs_light` is getting heavy and could use a refactor I think. Not sure how this is generally implemented in go however.